### DISCUSSION TITLE
Tweepy stream filter only following one filter

### DISCUSSION THREAD

**Discussion by gregyjames** [tweepy/tweepy#1698] [Questions] [answered]
I have python code that looks like the following:
`stream.filter(follow=[twitter_id], track=[hashtag_to_track], threaded=True)`

I need it to follow a certain user (myself) and get tweets live containing a certain hashtag (for example #trade). Twitter_id is a string that contains the numerical user id of a user and hashtag_to_track is a string that contains the hashtag to track. However, this code is not working. When I include both items in the filter it is getting all tweets including the hashtag not just specific tweets from the user. How can I restrict this to get only the tweets including the hashtag from the user specified? The twitter API for the POST statuses/filter endpoint says that:

> The track, follow, and locations fields should be considered to be combined with an OR operator. `track=foo&follow=1234` returns Tweets matching "foo" OR created by user 1234.

Is it possible to use this operator to combine the fields with Tweepy? Is there any other way to accomplish this with Tweepy?
Thank you!

**Comment by Harmon758** [comment, marked as answer]
> Is it possible to use this operator to combine the fields with Tweepy?

The endpoint ORs them by default. Tweepy simply passes the parameters to the endpoint.

> Is there any other way to accomplish this with Tweepy?

No, you'll have to filter the hashtag manually by checking for it in the Tweet.
